script/dump_templates

```diff
@@ -174,7 +179,7 @@ my $client = OpenQA::Client->new(apikey => $options{'apikey'}, apisecret => $opt
 
 my %result;
 
-for my $table (qw/Machines TestSuites Products JobTemplates/) {
+for my $table (@all_tables) {
```

Comment:
I don't get why iterating through `@all_tables` instead of `keys %tables`. Sure, performance is not an issue here, but I just spend few minutes looking at this snippet.